### ISSUE TITLE
Not installing packages correctly

### ISSUE THREAD

**Issue by nozpheratu** [chocolatey/chocolatey-cookbook#73]
Using version 0.6.1, I attempted to install jre8 as well as jruby with the following:

```ruby
include_recipe 'chocolatey'

chocolatey 'jre8'
chocolatey 'jruby'
```

But this fails with the following log output:
```
STDOUT: Installing the following packages:  
jruby  
By installing you accept licenses for the packages.  
 
jruby v9.0.4.0  
Downloading jruby 64 bit  
from 'http://jruby.org.s3.amazonaws.com/downloads/9.0.4.0/jruby_windows_x64_9_0_4_0.exe'  
Installing jruby...  
no jre found  
[ERROR] Running C:\Windows\TEMP\chocolatey\jruby\9.0.4.0\jrubyInstall.exe with   
-q  was not successful. Exit code was '83' Error Message:   
.  
At C:\ProgramData\chocolatey\helpers\functions\Start-ChocolateyProcessAsAdmin.p  
s1:92 char:5  
+     throw $errorMessage  
+     ~~~~~~~~~~~~~~~~~~~  
+ CategoryInfo          : OperationStopped: ([ERROR] Running...or Message:   
.:String) [], RuntimeException  
+ FullyQualifiedErrorId : [ERROR] Running C:\Windows\TEMP\chocolatey\jruby   
\9.0.4.0\jrubyInstall.exe with -q  was not successful. Exit code was '83'     
Error Message:   
.  
The install of jruby was NOT successful.  
Error while running 'C:\ProgramData\chocolatey\lib\jruby\tools\chocolateyInstall.ps1'.  
See log for details.  
 
Chocolatey installed 0/1 package(s). 1 package(s) failed.  
See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).  
Failures:  
- jruby 
STDERR:  
---- End output of "C:\ProgramData\chocolatey/bin/choco" install -y  jruby ---- 
Ran "C:\ProgramData\chocolatey/bin/choco" install -y  jruby returned 1 
```

Upon inspection on the remote Windows Server 2012 R2 machine, I can see that the `Path` and `JAVA_HOME` env variables are being set correctly with a path like `C:\Program Files\Java\jre1.8.0_66\bin;`. But oddly enough, that path doesn't even exist on the filesystem. There's no "Java" directory anywhere to be found!

RDPing into the machine and running `choco install jre8 --force` manually on Powershell seems to resolve things for me.

I should also note that the `Chocolatey::Helpers` module runs into an undefined method error for `powershell_out!`. I added a hotfix on my end by manually including the `Chef::Mixin::PowershellOut` module which seems to fix it.

**Comment by ferventcoder** [comment]
@mwrock the powershell_out! I thought was fixed in a newer cookbook version, can you verify?

**Comment by nozpheratu** [comment]
@ferventcoder Is the fix on supermarket?

**Comment by mwrock** [comment]
No. It sounds like you have the latest cookbook version, Can you tell me what version of the chef client you are using?

**Comment by nozpheratu** [comment]
@ferventcoder I verified that I'm definitely using the most up-to-date version. I'm using the Chef 12.2.1 client with OpsWorks. Here's the failure log: https://gist.github.com/nozpheratu/236a40b4f6026269edc9

**Comment by mwrock** [comment]
I have a feeling `powershell_out!` as its implemented here may not be friendly post v12.3.1. I'll confirm and make sure this gets fixed soon.

**Comment by mwrock** [comment]
you can find the fix in the supermarket 0.6.2. Thanks for reporting!!

**Comment by nozpheratu** [comment]
@mwrock Works as advertised. Also switching from the `jre8` to the `jdk8` package resolved my other issue I was having.
